charts/kube-prometheus-stack/hack/sync_grafana_dashboards.py

```diff
@@ -55,7 +55,7 @@ def new_representer(dumper, data):
     'node-rsrc-use': ' .Values.nodeExporter.enabled',
     'node-cluster-rsrc-use': ' .Values.nodeExporter.enabled',
     'nodes': ' .Values.nodeExporter.enabled',
-    'nodes-darwin': ' .Values.nodeExporter.enabled',
+    'nodes-darwin': ' .Values.nodeExporterMacOS.enabled',
```

Comment:
Give me some time to think, but I would rather we have some new property under nodeExporter than nodeExporterMacOS

Reply:
How about nodeExporterDarwin?

Reply:
I think I would prefer to have .Values.nodeExporter.enabled and something like .Values.prometheus-node-exporter.operatingSystemFlavors (with a list containing both linux and darwin for now). That way we could eventually add windows also.


@monotek or @GMartinez-Sisti could I have your opinion?

Reply:
Do you want me to do it as suggested by @GMartinez-Sisti?

Reply:
Let's do it 😄

Reply:
https://github.com/prometheus-community/helm-charts/pull/3778